docs/device-agent/install/device-agent-installer.md

```diff
@@ -189,6 +189,39 @@ sc.exe stop flowfuse-device-agent-<port>
 sc.exe query flowfuse-device-agent-<port>
 ```
 
+
+### Check the Device Agent service status
+
+#### Linux (systemd)
```

Comment:
Can you add a small 1/2 liner in here (above the "Linux" header") as to what the `status` command actually tells you please? Why would a user need to use it? What does it help with?